### PR TITLE
French translations for scheduler

### PR DESCRIPTION
**Pull request by xgehl**
Add missing French translations for scheduler

### CODE REVIEW

**Comment by Felx-B** [comment]
Is there any update on this ? What prevents this from being merged ?
@tsvetomir

**Comment by tsvetomir** [comment]
That would be my fault. I've lost track of the notifications for a while, sorry!

There's a similar suggestion for French translations in #104 by @GuillaumeMante that got merged earlier. This is a summary of the conflicting changes. I need your help to figure out which one is better.

```diff
diff --git a/messages/scheduler/scheduler.fr-FR.yml b/messages/scheduler/scheduler.fr-FR.yml
index e2d1184..3bc397e 100644
--- a/messages/scheduler/scheduler.fr-FR.yml
+++ b/messages/scheduler/scheduler.fr-FR.yml
@@ -101,7 +101,7 @@ kendo:
     allDay: toute la journée
 
     # The All events text displayed in the timeline views when there is no vertical grouping.
-    allEvents: Tous les événements
+    allEvents: Tous les évènements
 
     # The text of today's date in the header of the Calendar.
     calendarToday: AUJOURD'HUI
@@ -140,10 +140,10 @@ kendo:
     deleteDialogTitle: Suppression de l'élément
 
     # The text of the 'Delete current occurrence' button displayed in the scheduler recurring confirmation dialog.
-    deleteOccurrence: Suppression l'occurrence courante
+    deleteOccurrence: Supprimer l'occurence courante
 
     # The text similar to 'Do you want to delete only this event occurrence or the whole series?' displayed in the scheduler recurring confirmation dialog.
-    deleteRecurringConfirmation: Voulez-vous supprimer seulement cette occurrence ou toute la série ?
+    deleteRecurringConfirmation: Voulez-vous supprimer seulement cette occurence ou toute la série ?
 
     # The title of the recurring remove confirmation dialog, similar to 'Delete Recurring Item'
     deleteRecurringDialogTitle: Voulez-vous supprimer seulement cet évènement ou toute la série ?
@@ -218,7 +218,7 @@ kendo:
     recurrenceEditorEndOccurrence: occurrence(s)
 
     # The text similar to 'On' displayed in the scheduler recurrence editor.
-    recurrenceEditorEndOn: On
+    recurrenceEditorEndOn: Le
 
     # The text similar to 'Daily' displayed in the scheduler recurrence editor.
     recurrenceEditorFrequenciesDaily: Une fois par jour
@@ -272,7 +272,7 @@ kendo:
     recurrenceEditorWeekdaysWeekday: jour de la semaine
 
     # The text similar to 'Weekend Day' displayed in the repeat by section of the monthly recurrence pattern.
-    recurrenceEditorWeekdaysWeekendday: jour du week-end
+    recurrenceEditorWeekdaysWeekendday: Jour du week-end
 
     # The text similar to 'week(s)' displayed in the scheduler recurrence editor.
     recurrenceEditorWeeklyInterval: semaine(s)
@@ -305,13 +305,13 @@ kendo:
     timeHeader: Heure
 
     # The timeline month view title.
-    timelineMonthViewTitle: Mois
+    timelineMonthViewTitle: Timeline Mois
 
     # The timeline view title.
-    timelineViewTitle: Jour
+    timelineViewTitle: Timeline
 
     # The timeline week view title.
-    timelineWeekViewTitle: Semaine de travail
+    timelineWeekViewTitle: Timeline Semaine
 
     # The today button text displayed in the navigation.
     today: Aujourd'hui
```

**Comment by xgehl** [comment]
I updated this PR to take into account the other modifications and conflicts ;)

File: messages/scheduler/scheduler.fr-BE.yml [hunk 1]
```diff
@@ -4,13 +4,13 @@ kendo:
     dailyInterval: jour(s)
 
     # The text similar to 'Repeat every' displayed in the recurrence editor.
-    dailyRepeatEvery: 'Répéter chaque:'
+    dailyRepeatEvery: 'Répéter chaque :'
```
Comment (tsvetomir) [inline messages/scheduler/scheduler.fr-BE.yml:7]: Is it common to have a space before the colon?

Comment (xgehl) [reply messages/scheduler/scheduler.fr-BE.yml:7]: In French, yes.

**Comment by xgehl** [comment]
Is there any chance to get a 1.20.0 release (not dev I mean) ? @tsvetomir

**Comment by tsvetomir** [comment]
We'll grab #119 and release the official version tomorrow.

**Pull request merged**